samples/multiplatform/build.gradle

```
@@ -0,0 +1,42 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.61'
+}
+repositories {
+    mavenCentral()
+}
+group 'com.example'
+version '0.0.1'
+
+apply plugin: 'maven-publish'
+
+kotlin {
+    jvm()
+    js {
+        browser {
+        }
+        nodejs {
+        }
+    }
+    // For ARM, should be changed to iosArm32 or iosArm64
+    // For Linux, should be changed to e.g. linuxX64
+    // For MacOS, should be changed to e.g. macosX64
+    // For Windows, should be changed to e.g. mingwX64
+    mingwX64("windows")
+    sourceSets {
+
+        jvmTest {
+            dependencies {
+                //implementation kotlin('test')
+                //implementation kotlin('test-junit')
+                implementation("org.junit.jupiter:junit-jupiter-api:5.3.1")
+                implementation("ch.tutteli.atrium:atrium-fluent-en_GB-jvm:0.9.0-alpha2")
```

jvm modules don't have the -jvm suffix
```suggestion
                implementation("ch.tutteli.atrium:atrium-fluent-en_GB:0.9.0-alpha2")
```